comp/netflow/forwarder/component.go

```diff
@@ -0,0 +1,54 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
I wanted to use `fx` in the tests, so the server gets created by e.g.
```go
	server := fxutil.Test[Component](t, fx.Options(
		log.MockModule,
		....<other modules and fx options>...,
		Module,))
```
but if the server gets the demux as an injection then it breaks the pattern in the integration tests where we need to provide a mock forwarder. So I split it into a separate component so that the integration tests could specify the forwarder mock module and get a mock forwarder instead of the default forwarder that comes with the demux.